lualib/resty/kong/peer_conn.lua

```diff
@@ -0,0 +1,47 @@
+local ffi              = require "ffi"
+local C                = ffi.C
+local base             = require "resty.core.base"
+local ffi_str          = ffi.string
+local orig_get_request = base.get_request
+local get_phase        = ngx.get_phase
+local error            = error
+local NGX_ERROR        = ngx.ERROR
```

Comment:
Could we adjust the order of these variables? Now they looks a little mess.

Reply:
updated